team-builder/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import "./App.css";
+import Card from "./Component/Card";
+import Form from "./Component/Form";
+
+function App() {
+  const [card, newCard] = useState([
+    {
+      name: "Jacob Williams",
+      email: "jawilliamsdev@gmail.com",
+      role: "Full-stacks Web Dev"
+    }
+  ]);
+
+  return (
+    <div className="App">
+      <Form card={card} newCard={newCard} />
+      <Card card={card} />
```

Comment:
not really a big issue for a project of this size, but Cards or Team might be a better naming convention here.